src/components/Layout/Navbar/components/userProfile.module.css

```diff
@@ -1,11 +1,33 @@
 .userprofile_container {
-    display: flex;
-    gap: 10px;
-    align-items: center;
-    cursor: pointer;
+  display: flex;
+  gap: 10px;
+  align-items: center;
+  cursor: pointer;
 }
 
 .userprofile_user__first_name {
-    font-size: 1rem;
-    font-weight: 700;
+  font-size: 1rem;
+  font-weight: 700;
+}
+
+.userprofile_image__wrapper {
+  display: flex;
+  align-items: center;
+}
+
+.userProfile__dropdown_icon {
+  font-size: 25px;
```

Comment:
Can you create css variable for font size as it can be reused

Reply:
we are not using this icon that's for pointing out im removing this CSS